__tests__/app.test.js

```diff
@@ -122,15 +122,67 @@ describe("/api/reviews/:review_id", () => {
         expect(body.msg).toBe("review not found");
       });
   });
+  test("GET - 400 Id is not a number", () => {
+    return request(app)
+      .get(`/api/reviews/bananas`)
+      .expect(400)
+      .then((res) => {
+        expect((res) => {
+          expect(res.body.msg).toBe("Id is not a number");
+        });
+      });
+  });
+});
+
+describe("/api/reviews/:review_id/comments", () => {
+  test("GET - 200, responds with array of comments for given review_id", () => {
+    const reviewId = 3;
+    return request(app)
+      .get(`/api/reviews/${reviewId}/comments`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toHaveLength(3);
+        body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            review_id: expect.any(Number),
```

Comment:
Because we know that we are getting a specific review (you are using review id 3). We can test with hard coded test data